.github/CODEOWNERS

```diff
@@ -1,9 +1 @@
-* @joshuajbouw
-/engine/ @aleksuss
-/engine-precompiles/ @aleksuss
-/engine-sdk/ @aleksuss
-/engine-standalone-storage/ @RomanHodulak
-/engine-standalone-tracing/ @RomanHodulak
-/engine-test-doubles/ @joshuajbouw
-/engine-tests/ @joshuajbouw
-/engine-transactions/ @aleksuss
+* @aleksuss
```

Comment:
Only me? 😄 I would propose such a format:
```
* @aleksuss
/engine/ @birchmd
/engine-hashchain/ @birchmd
/engine-modexp/ @birchmd 
/engine-precompiles/ @birchmd
/engine-sdk/ @mrLSD 
/engine-standalone-storage/ @birchmd
/engine-standalone-tracing/ @birchmd
/engine-test-connector/ @mrLSD
/engine-test-doubles/ @mrLSD
/engine-tests/ @mrLSD 
/engine-transactions/ @mrLSD
/engine-workspace/ @mrLSD
```
If you guys don't mind.

Reply:
Yes, that looks ok to me.